cross/emby/Makefile

```diff
@@ -0,0 +1,24 @@
+PKG_NAME = emby
+PKG_VERS = 3.0.5724.5
+PKG_EXT = zip
+PKG_DIST_NAME = MediaBrowser-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/MediaBrowser/MediaBrowser.Releases/raw/master/Server
+PKG_DIR = emby
```

Comment:
`$(PKG_NAME)`. 

Btw, was MediaBrowser the old name or something? Otherwise I'd assume you use that for PKG_NAME etc.